### PR TITLE
PIS validation

### PR DESCRIPTION
**Pull request by alanraso**
PIS validation based on [this](http://www.macoratti.net/alg_pis.htm) algorithm.

+ Algorithm validating PIS
+ Tests and README following the pattern of other `is-valid-(...)` packages

### CODE REVIEW

File: packages/is-valid-pis/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  const weightedChecksum: number = generateChecksum(cleanPis.substr(0, length - 1), WEIGHTS);
+  const verifyingDigit: number = +cleanPis.charAt(length - 1);
+  const calculatedDigit: number = 11 - (weightedChecksum % 11);
+
+  return calculatedDigit === verifyingDigit
+    || (calculatedDigit === 10 && verifyingDigit === 0)
+    || (calculatedDigit === 11 && verifyingDigit === 0);
+}
+
+function removeSeparators(text: string): string {
```
Comment (hyanmandian) [inline packages/is-valid-pis/src/index.ts]: it's better to move this function above exported function (just to make reading like newspaper)

File: packages/is-valid-pis/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  return text.replace(/[ \(\).,*-]/g,  '');
+}
+
+function hasOnlyNumbers(text: string): boolean {
```
Comment (hyanmandian) [inline packages/is-valid-pis/src/index.ts]: it's better to move this function above exported function (just to make reading like newspaper)

File: packages/is-valid-pis/src/index.ts (hunk 1 above)
Comment (alanraso) [reply packages/is-valid-pis/src/index.ts]: Sure! Commit incoming...

File: packages/is-valid-pis/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+# UMD
+<script type='text/javascript' src='https://unpkg.com/@brazilian-utils/is-valid-pis/dist/index.umd.js'></script>
+```
+
+## Usage
+
+```js
+import { isValidPis } from '@brazilian-utils/is-valid-pis';
+
+isValidCpf('12081636699'); // true
```
Comment (hyanmandian) [inline packages/is-valid-pis/README.md]: You need to change this function to `isValidPis`

File: packages/is-valid-pis/src/tests/index.test.ts [hunk 4]
```diff
@@ -0,0 +1,67 @@
+import { BLACKLIST, PIS_LENGTH } from '../constants';
+import { isValidPis } from '../index';
+
+describe('isValidPis', () => {
+  describe('should return false', () => {
+    test('when it is on the BLACKLIST', () => {
+      BLACKLIST.forEach(cpf => expect(isValidPis(cpf)).toBe(false));
```
Comment (hyanmandian) [inline packages/is-valid-pis/src/tests/index.test.ts]: you need to change cpf to pis in the arrow function

**Comment by hyanmandian** [comment]
Awesome job @alanraso, just resolve conflicts and I will merge your PR :+1:

**Comment by alanraso** [comment]
Hey, @hyanmandian, thank you for your support!
I resolved the conflict with rebase, hope that's not a problem.

**Pull request merged**